### DISCUSSION TITLE
Question.

### DISCUSSION THREAD

**Discussion by Vhirttuegaming** [3kh0/ext-remover#1151] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Why does Lt Meat only temporarily work on Light-speed filter meaning the filter would work again in 30 minutes. And is there a permanent solution?

**Comment by AshtonDavies** [comment, marked as answer]
It does not work as well anymore. I recommend switching to Dextensify or a different kind of exploit.
